kotlin-analysis-clones/src/test/kotlin/org/jetbrains/research/ml/kotlinAnalysis/CloneDetectionAdapterTest.kt

```diff
@@ -2,9 +2,9 @@ package org.jetbrains.research.ml.kotlinAnalysis
 
 import com.intellij.psi.util.PsiTreeUtil
 import org.jetbrains.kotlin.psi.KtNamedFunction
-import org.jetbrains.research.ml.kotlinAnalysis.util.Extension
-import org.jetbrains.research.ml.kotlinAnalysis.util.ParametrizedBaseTest
-import org.jetbrains.research.ml.kotlinAnalysis.util.getPsiFile
+import org.jetbrains.research.pluginUtilities.util.Extension
```

Comment:
Maybe we can either use import `org.jetbrains.research.pluginUtilities.util.Extension` in our package `org.jetbrains.research.ml.kotlinAnalysis.util` or move `VirtualFile.isKotlinRelatedFile()` there?

Reply:
What about the third option: use `org.jetbrains.research.pluginUtilities.util.Extension` in `isKotlinRelatedFile` function?) and delete `Extension` class from our repo

Reply:
Yes, sure! I failed to explain it, but this was my idea =)

Reply:
I used in range analysis Extension.CSV, can you add it to utilities either, please?